kitazb_hw03.py

```diff
@@ -0,0 +1,190 @@
+#################################################################################
+# Author: Besher Kitaz
+# Username: kitazb
+#
+# Assignment: HW03
+# Purpose: Drawing a house with Turtle library
+#
+#
+#################################################################################
+# Acknowledgements:
+#
+# Document link: https://docs.google.com/document/d/13fdydW8mxVLdYfAWMG7x2zIYqjoSEVAWuOItJJ7VOT4/edit
+#################################################################################
+
+import turtle
+
+
+def draw_house_body(t):
+    """
+    draws a body for the house
+    :args: a turtle object
+    :returns:none
+    """
+    t.fillcolor('green')
+    t.begin_fill()
+    for i in range(4):
+        t.forward(250)
+        t.left(90)
+    t.end_fill()
+
+
+def draw_rooftop(t):
+    """
+    draws a rooftop for the house
+    :args: a turtle object
+    :returns:none
+    """
+    t.fillcolor('red')
+    t.begin_fill()
+    t.right(45)
+    t.forward(176)
+    t.right(90)
+    t.forward(176)
+    t.end_fill()
+
+
+def draw_window_square(t):
+    """
+        gets a turtle object to draw a square shape for the window
+        :args: t, turtle object, px, for the position of the window horizontally, and py for vertical position
```

Comment:
Your docstring doesn't match your function.

Reply:
Oh, sorry. I will fix it. This is because I changed the function, but apparently forgot to change this.

Reply:
Should be fixed now.